pkg/languages/defaults/gradle.go

```diff
@@ -0,0 +1,67 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/draft/pkg/reporeader"
+	log "github.com/sirupsen/logrus"
+)
+
+type GradleExtractor struct {
+}
+
+// GetName implements reporeader.VariableExtractor
+func (*GradleExtractor) GetName() string {
+	return "gradle"
+}
+
+// MatchesLanguage implements reporeader.VariableExtractor
+func (*GradleExtractor) MatchesLanguage(lowerlang string) bool {
+	return lowerlang == "gradle" || lowerlang == "gradlew"
+}
+
+// ReadDefaults implements reporeader.VariableExtractor
+func (*GradleExtractor) ReadDefaults(r reporeader.RepoReader) (map[string]string, error) {
+	extractedValues := make(map[string]string)
+	files, err := r.FindFiles(".", []string{"*.gradle"}, 2)
+	if err != nil {
+		return nil, fmt.Errorf("error finding gradle files: %v", err)
+	}
+	if len(files) > 0 {
+		f, err := r.ReadFile(files[0])
+		if err != nil {
+			log.Warn("Unable to read build.gradle, skipping detection")
```

Comment:
Thanks for catching this, totally forgot about this. I changed it for some testing and never put back.

Reply:
@davidgamero it would be *.gradle , they can name the file anything, but the extension would ```.gradle```. I will change the logger statement to "unable to read gradle file in your repository, skipping detection" , ?

Reply:
okay sounds good